### PR TITLE
Update the Getting Started pages

### PR DESCRIPTION
**Pull request by hans-notifi**
(Note that this PR is descended from #17)

- Remove the old "quickstart" page
- Instead, include a guide for signing up for Notifi, setting up a notification topic, setting up a React card, adding that React card to your website, and sending messages. 
- Do this for all three message types (Community Manager, self-hosted, and Notifi-hosted).

### CODE REVIEW

**Comment by hans-notifi** [comment]
Note that the text is ready for review, but we should add some screenshots.

**Comment by hans-notifi** [comment]
Should I add screenshots to these docs right away?

File: docs/getting-started.md [hunk 1]
```diff
@@ -2,20 +2,267 @@
...
+## Setting Up A Notification Topic
+
+Notifi has three types of notifications:
+
+1. Messages written in Community Manager in Admin Portal
+2. Messages triggered by your server through the Notifi SDK
+3. Messages triggered by Notifi-hosted code, usually in response to on-chain events
+
+For this Getting Started guide, we will be creating a Community Manager topic.
+Check our our
```
Comment (nimeshamin) [inline docs/getting-started.md]: out

File: docs/getting-started.md [hunk 2]
```diff
@@ -2,20 +2,267 @@
...
+  NotifiSubscriptionCard,
+} from '@notifi-network/notifi-react-card';
+import '@notifi-network/notifi-react-card/dist/index.css';
+```
+
+and paste this code at the exact spot where you want your card to be:
+```
+<NotifiContext
+    dappAddress="<YOUR OWN DAPP ADDRESS HERE>"
+    env="Production"
```
Comment (nimeshamin) [inline docs/getting-started.md]: Let's make env optional and default to Production

File: docs/getting-started.md [hunk 3]
```diff
@@ -2,20 +2,267 @@
...
+
+## Sending Your First Message
+
+Finally, now that we have a topic and a user signed up to it,
+it's time to send a message!
+
+1. Navigate to the **Community Manager** page in Admin Portal.
+2. Under **Audience/Notification Topic**, select the Community Manager
+   topic you've created.
+3. Enter some text in the **Campaign Name** field. Your audience won't
```
Comment (nimeshamin) [inline docs/getting-started.md:173]: Terms and jargon that we use should ideally link to some page that explains in much more depth. One example here would be linking this Campaign Name to a "Using Community Manager" or something like that section where it goes into depth

File: docs/getting-started-with-self-hosted.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+const MY_SID = process.env.MY_SID;
+const MY_SECRET = process.env.MY_SECRET;
+const EVENT_TYPE_ID = process.env.MY_EVENT_TYPE_ID;
+
+const gqlClient = createGraphQLClient("Production");
+const notifiService = createNotifiService(gqlClient);
+const dataplaneApiClient = createDataplaneClient("Production");
+const client = new NotifiClient(notifiService, dataplaneApiClient);
+
+const { token, expiry } = await client.logIn({
```
Comment (nimeshamin) [inline docs/getting-started-with-self-hosted.md:130]: do we want to give them boilerplate for using the refresh jwt?

File: docs/getting-started-with-self-hosted.md [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+and our [Dataplane REST API](./alert-trigger/dataplane-api/).)
+
+1. Install [Node.js](https://nodejs.org/en) version 18 or later.
+2. Run `$ npm init` in an empty directory to create your project
+3. Run `$ npm install @notifi-network/notifi-node` to add the [Notifi Node.js SDK](https://www.npmjs.com/package/@notifi-network/notifi-node)
+   to your project.
+4. Add `"type": "module"` to your project's `package.json`, since we will be using ES6 modules and top-level `await`
+5. Create a `main.js` file with the following code:
+
+```javascript
```
Comment (nimeshamin) [inline docs/getting-started-with-self-hosted.md:107]: Shoudl we also put a link here to link to the code/examples in our sdk repo?

File: docs/getting-started.md (hunk 2 above)
Comment (hans-notifi) [reply docs/getting-started.md]: Done: https://github.com/notifi-network/notifi-sdk-ts/pull/537

Comment (hans-notifi) [reply docs/getting-started.md]: I'm going to leave it in until we have a release that actually allows `env` to be optional, though.

File: docs/getting-started.md (hunk 3 above)
Comment (hans-notifi) [reply docs/getting-started.md:173]: Agreed. I've added most of the "check out more stuff" content at the bottom under the "Next Steps" section, but I also added a link here to a stub about our reporting features.

File: docs/getting-started-with-self-hosted.md (hunk 4 above)
Comment (hans-notifi) [reply docs/getting-started-with-self-hosted.md:130]: Unfortunately, those functions haven't been added to the Notifi SDK, so I can't. I believe Eric's long term plan with the SDK is to manage the tokens internally.

File: docs/getting-started-with-self-hosted.md (hunk 5 above)
Comment (hans-notifi) [reply docs/getting-started-with-self-hosted.md:107]: Yes. I ended up reworking this section to only include code samples for EVM setup, and added a link in an admonition section to view examples for other chains.

**Review by bruno-notifi — APPROVED** [review]
lgtm! I am still reading pages from 17.

**Review by gustavo-notifi — APPROVED** [review]

**Pull request merged**